internal/bminventory/inventory.go

```diff
@@ -95,6 +95,34 @@ sudo journalctl -u agent.service
 **  **  **  **  **  **  **  **  **  **  **  **  **  **  **  **  **  ** **  **  **  **  **  **  **
 `
 
+const redhatRootCA = `
+-----BEGIN CERTIFICATE-----
```

Comment:
I don't think the certificate should be in github, for everyone to see

Reply:
It is available from https://password.corp.redhat.com/RH-IT-Root-CA.crt without VPN